bashbot/bot.py

```diff
@@ -65,7 +65,10 @@ async def on_message(self, message: Message):
 
         if self.is_invoke(message):
```

Comment:
This is checking if the message is starting with prefix + dot so inside "elif" you will never have a message with prefix + dot.